cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -56,6 +56,9 @@ describe('Script Builder', () => {
   const selectSchema = () => {
     cy.log('select bucket')
     selectBucket(bucketName)
+    cy.getByTestID('flux-editor').contains(`from(bucket: "${bucketName}")`, {
+      timeout: 30000,
+    })
```

Comment:
LSP is now stateful. So it needs to update the bucket in it's state, before the next update. Cypress is much faster than human clicking --> so check for the editor text update.